<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontListDialog</class>
 <widget class="QDialog" name="FontListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenSCAD Font List</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>16</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="4" column="4">
    <widget class="QPushButton" name="okButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="copyButton">
     <property name="toolTip">
      <string>Paste font selector to Editor Window</string>
     </property>
     <property name="text">
      <string>Copy to Clipboard</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="filterLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Filter:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This list shows the fonts currently registered with OpenSCAD.&lt;/p&gt;&lt;p&gt;Example:&lt;/p&gt;&lt;pre style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;  text(t = &amp;quot;OpenSCAD&amp;quot;, font = &amp;quot;DejaVu Sans&amp;quot;);&lt;/span&gt;&lt;/pre&gt;&lt;pre style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;  text(t = &amp;quot;OpenSCAD&amp;quot;, font = &amp;quot;Liberation Sans:style=Italic&amp;quot;);&lt;/span&gt;&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../openscad.qrc"/>
 </resources>
 <connections/>
</ui>
